mathcomp/Makefile.common

```diff
@@ -64,7 +76,7 @@ install: __always__ Makefile.coq
 # counting lines of Coq code -----------------------------------------
 .PHONY: count
 
-COQFILES := $(shell grep '.v$$' Make)
+COQFILES = $(shell grep '.v$$' $(COQPROJECT))
```

Comment:
@CohenCyril @SkySkimmer wrt #238 I think this line is fishy. `$(COQPROJECT)` points to `_CoqProject` that, IIRC, only contains `-R` directives but not the list of files.

Reply:
I believe it points to https://github.com/math-comp/math-comp/blob/d6dc5741ba44808e5f2f01a238d972ec2c11737f/mathcomp/fingroup/Make (which does have v files) as per https://github.com/math-comp/math-comp/blob/d6dc5741ba44808e5f2f01a238d972ec2c11737f/mathcomp/fingroup/Makefile#L3

Reply:
> [...] as per
> 
> [math-comp/mathcomp/fingroup/Makefile](https://github.com/math-comp/math-comp/blob/d6dc5741ba44808e5f2f01a238d972ec2c11737f/mathcomp/fingroup/Makefile#L3)
> 
> Line 3 in [d6dc574](/math-comp/math-comp/commit/d6dc5741ba44808e5f2f01a238d972ec2c11737f)
> 
>  COQPROJECT="Make"

Yes indeed, but... could the double quotes be guilty here?

Reply:
No, it gets sent to the shell which will remove them.